cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -1880,9 +1878,7 @@ func init() {
 	testNamespaces = flagSet.String("namespaces", "eth,web3,net,debug", "Comma separated list of rpc namespaces to test")
 	testFuzz = flagSet.Bool("fuzz", false, "Flag to indicate whether to fuzz input or not.")
 	testFuzzNum = flagSet.Int("fuzzn", 100, "Number of times to run the fuzzer per test.")
-	seed = flagSet.Int64("seed", 123456, "A seed for generating random values within the fuzzer")
 
-	rand.Seed(*seed)
```

Comment:
not being used? i guess this isn't useful for anything. so removing.

Reply:
This is used. Do not remove it.

Reply:
i'm a little confused. does it set the seed for the `rand` usage in argfuzz?